README.md

```diff
@@ -1,6 +1,5 @@
 # Library for python loading of Synthesis AI dataset. 
 Documentation: https://synthesis-ai-dev.github.io/face_api_dataset/html/index.html
 
-See https://github.com/Synthesis-AI-Dev/face_api_dataset/blob/main/example.ipynb for jupyter notebook with tutorial.
-
-You can also see this tutorial in colab: https://colab.research.google.com/drive/1TH_UDHWQ2-6b_mZCfJjgcRtMVleLuRVd?usp=sharing.
+See a [quickstart](https://github.com/Synthesis-AI-Dev/face_api_dataset/blob/main/examples/quickstart.ipynb) for jupyter notebook with tutorial.
+Also visit a more detailed example with [landmarks](https://github.com/Synthesis-AI-Dev/face_api_dataset/blob/main/examples/landmarks.ipynb).
```

Comment:
Any reason to drop colab link?

Reply:
@adavydow it's not valid anymore. Already created an issue to setup a new one https://github.com/Synthesis-AI-Dev/face_api_dataset/issues/17